single_app.ipynb

```diff
@@ -78,17 +78,7 @@
    "source": [
     "app_base = AiidaLabApp(name, app_data, AIIDALAB_APPS)\n",
     "\n",
-    "try:\n",
-    "    display(AppManagerWidget(app_base, minimalistic=False))\n",
-    "except Exception as error:\n",
```

Comment:
Catching all exceptions like this is really bad, since it is hard to debug and users cannot provide stack traces in their bug reports.